stlab/concurrency/future.hpp

```diff
@@ -225,6 +225,19 @@ using reduced_t = typename reduced_<T>::type;
 template <typename T>
 struct reduction_helper;
 
+template <typename R>
+R&& reduce(R&& r) {
+    return std::forward<R>(r);
+}
+
+static inline auto reduce(future<future<void>>&& r) -> future<void>;
```

Comment:
I don't think `static` does anything useful here.

Reply:
Agree, I will propose a fix in the next patch set.